src/python/grpcio_tests/tests_aio/unit/init_test.py

```diff
@@ -11,56 +11,14 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-import asyncio
 import logging
 import unittest
 
-import grpc
 from grpc.experimental import aio
 from tests_aio.unit._test_server import start_test_server
 from tests_aio.unit._test_base import AioTestBase
 
 
-class TestAioRpcError(unittest.TestCase):
```

Comment:
Why wouldn't we want these tests anymore?

Reply:
They have been basically moved to the right file for having coherence between unit test and grpc modules, now it lives into the`channel_test.py` file. Maybe I would move the `Call`  and the `AioRpcError` implementation into a specific `call.py` file, just for not having a massive `_channel.py` file. If so, the tests would be moved to the `call_test.py` file.